src/css/components/help-menu.less

```diff
@@ -3,15 +3,26 @@
  * =============
  */
 .help-menu {
-  margin: @base-spacing-unit*2+1 @base-spacing-unit;
-  width: @base-spacing-unit*2.5;
-  padding: 0 @base-spacing-unit*2.5;
+  margin: 0;
+  padding: @base-spacing-unit*2 @base-spacing-unit*2;
   cursor: pointer;
-  color: @manatee;
+  color: @white;
   background-position: center left;
+  line-height: 1;
+  opacity: .4;
 
-  &.active {
-    color: @white;
+  .icon.help {
+    height: 18px;
+    width: 18px;
```

Comment:
It should be possible to achieve the same by removing this `.icon.help` section entirely from here, and by adding the class `icon-mini` to the `<i>` element added in the `helpMenu`. The advantages are:
1) we get rid of unnecessary "overrides" and the specificity issues associated with it
2) we re-use pre-existing utility classes
3) we don't introduce yet another icon size

What do you think?